### DISCUSSION TITLE
Spelling mistake in mobile app for iOS

### DISCUSSION THREAD

**Discussion by ghost** [community/community#45964] [Mobile] [answered]
### Select Topic Area

Bug

### Body

There is a spelling error inside the **code options settings** of the GitHub mobile app on iOS in the Spanish version, in the **Edit** section it says `mosrar`, instead it should say `mostrar` which  translates as show.
I add a screenshot where you can see the error, thanks!

![IMG_8156](https://user-images.githubusercontent.com/94091554/215910701-0d21f1e7-97f3-4b9b-ba79-cda65e7a898e.jpeg)

**Comment by alcere** [comment, marked as answer]
@j-cordz thanks reporting! I have created an issue and the team will fix the typo as soon as possible, muchas gracias!
